helm/values.yaml

```diff
@@ -47,6 +47,9 @@ manager:
     java:
       repository: public.ecr.aws/aws-observability/adot-autoinstrumentation-java
       tag: v1.31.1
+    python:
+      repository: ghcr.io/open-telemetry/opentelemetry-operator/autoinstrumentation-python
```

Comment:
Is there a reason we're using GHCR instead of ECR? Is it not available yet?

Reply:
Yes this is temporary until the ADOT python image is available.